models/nft/ethereum/nft_ethereum_loans.sql

```diff
@@ -0,0 +1,392 @@
+            get_json_object(loanDetail, '$.nftAsset') as collectionContract,
+            get_json_object(loanDetail, '$.nftTokenId') as tokenId,
+            get_json_object(loanDetail, '$.borrowAmount') as principal_raw,
+            get_json_object(loanDetail, '$.borrowAsset') as currency,
+            get_json_object(loanDetail, '$.repayAmount')  as repayment_raw,
+            get_json_object(loanDetail, '$.loanDuration') / 86400 as duration,
+            contract_address, loanId, 'v1' as version
+        from {{ source('xy3_ethereum','XY3_evt_LoanStarted') }}
+        union all
+        select 
+            evt_tx_hash, evt_block_time, borrower, lender,
+            get_json_object(loanDetail, '$.nftAsset') as collectionContract,
+            get_json_object(loanDetail, '$.nftTokenId') as tokenId,
+            get_json_object(loanDetail, '$.borrowAmount') as principal_raw,
+            get_json_object(loanDetail, '$.borrowAsset') as currency,
+            get_json_object(loanDetail, '$.repayAmount')  as repayment_raw,
+            get_json_object(loanDetail, '$.loanDuration') / 86400 as duration,
+            contract_address, loanId, 'v2' as version
+        from {{ source('xy3_ethereum','XY3_V2_evt_LoanStarted') }}
+    ) t
+),
+
+x2y2 as (
+    select l.*, r.evt_block_time as repay_time
+    from x2y2_base l left join (
+        select * from {{ source('xy3_ethereum','XY3_evt_LoanRepaid') }}
+        union all
+        select * from {{ source('xy3_ethereum','XY3_V2_evt_LoanRepaid') }}
+    ) r on l.loanId=r.loanId and l.contract_address=r.contract_address
+),
+
+-- arcade loans
+arcade_v1_deposits as (
+    select cast(bundleId as decimal) as bundleId, evt_block_time, tokenAddress as collectionContract, tokenId, 'v1' as version
+    from {{ source('pawnfi_ethereum','AssetWrapper_evt_DepositERC721') }} --- deposit NFTs to vault V1
+    union all
+    select cast(bundleId as decimal) as bundleId, evt_block_time, tokenAddress as collectionContract, tokenId, 'v1.2' as version
+    from {{ source('pawnfi_ethereum','AssetWrapperv102_evt_DepositERC721') }}
+),
+
+arcade_v1_base as (
+    select *, 365 * 1.0 / duration * (interest_raw * 1.0 / principal_raw) * 100 as apr
+    from (
+        select c.evt_tx_hash, evt_block_time, borrower, lender, c.contract_address, p.loanId, 'v1' as version,
+            get_json_object(terms, '$.collateralTokenId') as bundleId,
+            get_json_object(terms, '$.payableCurrency') as currency,
+            get_json_object(terms, '$.principal') as principal_raw,
+            get_json_object(terms, '$.interest') as interest_raw,
+            get_json_object(terms, '$.durationSecs') / 86400 as duration
+        from {{ source('pawnfi_ethereum','LoanCore_call_startLoan') }} p left join {{ source('pawnfi_ethereum','LoanCore_evt_LoanCreated') }} c on p.call_block_time=c.evt_block_time and p.call_tx_hash = c.evt_tx_hash
+        where call_success=true
+        union all

```

Comment:
Please start `left join` and `inner join` in the new line.
```suggestion
        from {{ source('pawnfi_ethereum','LoanCore_call_startLoan') }} p 
        left join {{ source('pawnfi_ethereum','LoanCore_evt_LoanCreated') }} c 
        on p.call_block_time=c.evt_block_time 
        and p.call_tx_hash = c.evt_tx_hash
        where call_success=true
        union all

```

Reply:
This would be a good habit in writing sql and it helps the review/compilation too.